lib/service.js

```diff
@@ -21,17 +21,31 @@ export default class WdioImageComparisonService extends BaseClass {
      * @param {Array.<Object>} capabilities list of capabilities details
      */
     before(capabilities) {
-        const instanceData = getInstanceData(capabilities)
+        if(typeof capabilities['browserName'] !== 'undefined') {
+            console.log('Adding to global browser')
+            this.addCommandsToBrowser(capabilities, browser)
+        } else {
+            const browserNames = Object.keys(capabilities)
+            console.log('Adding to multibrowser: ', browserNames)
+            for(const browserName of browserNames) {
+                this.addCommandsToBrowser(capabilities[browserName], global[browserName])
```

Comment:
To get all the proper data of the instance we need to change this and look into the `capabilities` of the `capabilities`. Then we have access to the capabilities of the current running instance

```suggestion
                this.addCommandsToBrowser(capabilities[browserName].capabilities, global[browserName])
```

Reply:
Ah yep, good point! Do i merge it through this interface or just include it in my next commit addressing the other issues?

Reply:
You can click on `Commit suggestion` or add it to your later, that's up to you